distribution/src/resources/drill-config.sh

```diff
@@ -334,6 +334,7 @@ fi
 # provided in drill-env.sh.
 
 export DRILL_PID_DIR=${DRILL_PID_DIR:-$DRILL_HOME}
+export GRACEFUL_FILE_SUFFIX=GRACEFUL_SIGFILE
```

Comment:
Where is the value for `GRACEFUL_SIGFILE` defined? Provide a default if not set
```
export FOO=${FOO:-"value"}
```

Reply:
I thought you asked me to change the name of the file graceful to GRACEFUL_SIGFILE. Let me change it.